requirements.txt

```diff
@@ -1,5 +1,5 @@
 PyYAML==5.1
 argparse==1.1
 msgpack-python==0.4.6
-gevent==1.1b5
+gevent==21.1.1
```

Comment:
NB: 21.1.2 is [out](http://www.gevent.org/changelog.html).

Reply:
Updated